src/guides/v2.3/coding-standards/docblock-standard-general.md

```diff
@@ -685,6 +691,58 @@ Specifically, this is possible when a PHP-file composed from multiple file inclu
  */
 ```
 
+### @method tag {#method}
+
+The `@method` allows a class to know which ‘magic’ methods are callable.
+
+Syntax:
+
+```bash
+@method [[static] return type] [name]([[type] [parameter]<, ...>]) [<description>]
+```
+
+[See](https://docs.phpdoc.org/references/phpdoc/tags/method.html) more information about `@method` tag.
+
+```php
+/**
+ * Image operations
+ *
+ * @method string getFile()
+ * @method string getLabel()
+ * @method string getPosition()
+ */
+class Image extends \Magento\Framework\Model\AbstractModel
+{
+    //.....
+}
+```
+
+### @link tag {#link}
+
+The `@link` tag indicates a custom relation between associated Structural Elements and a website, which is identified by an absolute URI.
+
+Syntax:
+
+```bash
+@link [URI] [<description>]
+```
+
+### @since tag {#since}
```

Comment:
This is not used in the original Magento repository. `since` tag is set automatically during Composer packages publication (this is not contributed back to the Git repo). So I would not have it as a recommendation for everybody as this may cause failure of publication tools if somebody adds this tag manually.

Reply:
@buskamuza I agreed, can I remove that or need to add any additional description? also please let me know any changes required I am happy to do that.

Reply:
@dobooth @lenaorobei @buskamuza did you have any chance to review this?

Reply:
@buskamuza as recommended, I have removed `@since` tag in commits. can you please review now.